deblur/workflow.py

```diff
@@ -576,6 +577,10 @@ def get_files_for_table(input_dir,
     file_end : string
         the ending of all the fasta files to be added to the table
         (default '.fasta.trim.derep.no_artifacts.msa.deblur.no_chimeras')
+    num_extra_name_chars : int (optional)
+        the number of additional characters to remove from the end of the
+        file name to get the sample name
+        default is len('.fasta')=6
```

Comment:
i think the problem is we don't know the sample name does not contain '.' in it.
i.e. say we start with a samples called "X.Y"
the reads file will be "X.Y.fasta"
and then the final deblurred file will be called "X.Y.fasta.trim.derep.ZZZ.nochimers"
we can use rsplit instead but i think this is more clear?